workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/PurchasedAppsDashboardTableRow.tsx

```diff
@@ -157,7 +174,7 @@ export function PurchasedAppsDashboardTableRow({
 									title={
 										orderStatusIsNotCompleted
 											? 'This order must be completed before downloading this app.'
-											: ''
+											: undefined
```

Comment:
When you leave an empty string here, we keep getting an error. So I changed it to undefined and that fixed it.

Reply:
It's weird because the the other case it's also a string 🤔 

But it's fine!